packages/core/src/utils/spanUtils.ts

```diff
@@ -81,6 +83,19 @@ export function spanToTraceHeader(span: Span): string {
   return generateSentryTraceHeader(traceId, spanId, sampled);
 }
 
+/**
+ *  Converts the span links array to a flattened version to be sent within an envelope
+ */
+export function convertSpanLinksForEnvelope(links: SpanLink[]): SpanLinkJSON[] {
```

Comment:
q: I assume this is getting tested in subsequent PRs when we serialize envelopes? If so, ignore me and no objections :)